### DISCUSSION TITLE
table组件是否可以增加类似tree组件的render-after-expand

### DISCUSSION THREAD

**Discussion by beishuangzz** [element-plus/element-plus#17627] [General]
在使用table组件渲染树形数据时, 未展开的节点的子节点也出现在dom中, 节点多的时候导致页面有点卡, 虽然可以通过懒加载解决, 但如果数据本身是同步的, 强行使用懒加载反而不方便

**Comment by elarbor** [comment]
[展开行](https://element-plus.org/zh-CN/component/table.html#%E5%B1%95%E5%BC%80%E8%A1%8C)怎么做懒加载啊？不是树形数据
![动画](https://github.com/user-attachments/assets/338858fd-1683-4431-9b22-020fa9cb68a3)
